Makefile

```diff
@@ -214,65 +219,61 @@ require-gh-token:
 	@if [[ -z "$$GH_TOKEN" ]]; then echo "GH_TOKEN environment value is required."; exit 1; fi
 
 require-secrets-dir:
-	if [[ -z "$$SECRETS_DIR" ]]; then echo "SECRETS_DIR environment value is required."; exit 1; fi
+	@if [[ -z "$$SECRETS_DIR" ]]; then echo "SECRETS_DIR environment value is required."; exit 1; fi
 
 require-secrets: require-secrets-dir
 	@if [[ -z "$$BNS_CERT_PASS" ]]; then echo "BNS_CERT_PASS environment value is required."; exit 1; fi
 
+$(BNS_CERT): require-secrets-dir
+	@cp -f $$SECRETS_DIR/$(BNS_CERT) $(BNS_CERT)
+
 require-lantern-binaries:
 	@if [[ ! -d "$(LANTERN_BINARIES_PATH)" ]]; then \
 		echo "Missing lantern binaries repository (https://github.com/getlantern/lantern-binaries). Set it with LANTERN_BINARIES_PATH=\"/path/to/repository\" make ..." && \
 		exit 1; \
 	fi
 
-docker-package-debian-386: require-version docker-linux-386
```

Comment:
These were with "debian" but when used as dependencies were named with "linux".  I renamed them using "linux" for consistency.